exercises/high-scores/.meta/solutions/high_scores.rb

```diff
@@ -0,0 +1,27 @@
+class HighScores
+  def initialize(scores)
+    @scores = scores
+  end
+
+  def scores
+    @scores
+  end
+
+  def highest
+    scores.max
+  end
+
+  def latest
+    scores.last
+  end
+
+  def top
+    scores.sort.reverse.take(3)
+  end
+
+  def goal
+    verdict = (latest == highest ? "" : " #{highest-latest} short of")
```

Comment:
`verdict` doesn't look like a great name for this variable. For me, verdict would be this whole sentence "That's#{verdict} your personal best!".

Do you have any other ideas? I even considered doing something like this:

```ruby
if latest == highest
  verdict = "That's #{highest - latest} short of your personal best!"
else
  verdict = "That's your personal best!"
end

"Your latest score was #{latest}. #{verdict}"
```

Or extracting it to a private method 😏

Reply:
Also thought about the possibility of changing `goal` to `verdict`, `latest_verdict` or `current_verdict`...

Reply:
I'm okay with how it is. 
This is how I solved it when I tested the tests:
```
if highest != latest 
      delta = " #{highest - latest} from "
    else 
      delta = " "
    end
    "Your latest score was #{latest}. That's #{difference} your personal best!".squeeze
```

Reply:
Yeah, `difference` is better. I'm going to commit that.